devicon.json

```diff
@@ -4136,6 +4136,30 @@
         "color": "#56c5a8",
         "aliases": []
     },
+    {
+        "name": "knexjs",
+        "altnames": [],
+        "tags": [
+            "database",
+            "query",
+            "sql"
```

Comment:
```suggestion
            "sql",
            "javascript"
```

Reply:
> I suggest adding a `javascript` tag Other than that this looks excellent 💯
> 
> Ohh and see if you can find a wordmark logo for this if possible. I found this on google: ![](https://camo.githubusercontent.com/e6e059808c39cbf62161a5cf55ec08ae2cf43aee855ea20bda4b73910cb046f1/68747470733a2f2f676f6f676c652e6769746875622e696f2f73716c636f6d6d656e7465722f696d616765732f6b6e65782d6c6f676f2e706e67) https://google.github.io/sqlcommenter/images/knex-logo.png Which looks similar enough to the wordmark logos on their website: ![image](https://user-images.githubusercontent.com/43886029/195909878-08c54754-bc74-4b58-a6ac-564519b5e5de.png)
> 
> Sadly it's a .png and not a .svg, so if you don't want to convert and add it (or make your own), that's okey too :)

This one is fully convertible to `.svg` by rasterization, as it's just simple shapes with solid colors and has a good resolution.

I'm just not sure which one I should choose, but I was inclined to choose the second one with "SQL query builder" written, because it's on their current website.

Reply:
Great news!
I'm not sure about having more wordmarks than just the technology name, but I'm not heavily against it either.
Feel free to choose the one you see fit :)